tests/python/unittest/test_contrib_intgemm.py

```diff
@@ -0,0 +1,205 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
for master branch, we intend to move most of the specialized ops to the `npx` space, and it would be great if you could register an alias there too and add a numpy version of the following tests.

Reply:
Done in 29cc970